doc/how-to-use.md

```diff
@@ -0,0 +1,400 @@
+## How to use multus-cni?
+
+### Prerequisite
+
+Kubelet must be configured to run with the CNI network plugin. Edit `/etc/kubernetes/kubelet` file and add `--network-plugin=cni` flags in `KUBELET\_OPTS `as shown below:
```

Comment:
On my system, the file `/etc/kubernetes/kubelet` does not exist. If this is the correct file, then add a comment that file may need to be created.

After I ran the 'cat ./images/{multus-daemonset.yml,flannel-daemonset.yml} | kubectl apply -f -' command, 'echo $KUBELET_OPTS' indicated the environmental variable wasn't set, so not sure it if took.

Reply:
Changed to indicate K8s document for CNI.